index.d.ts

```diff
@@ -0,0 +1,17 @@
+type decodeFuncType = (
```

Comment:
Project exports _three_ modules. Will `querystring/encode` and `querystring/decode` be also covered with this typings?

Reply:
It wasn't earlier but I have made change to accomodate that